textview.go

```diff
@@ -634,9 +646,6 @@ func (t *TextView) Write(p []byte) (n int, err error) {
 		}()
 	}
 
-	t.Lock()
```

Comment:
Part of the slowness I had been experiencing was that there are actually two locks here, but they don't appear to be necessary (it's just skipping over a single branch, might as well keep it locked unless we expect / want things to change between checking if changed should be called and doing the actual write, but I don't see why we'd want that)